network_modeling/models/cray-network-architecture.yaml

```diff
@@ -191,6 +204,11 @@ network_v2_tds:
         - name: "spine"
           speed: 10
           number: 2
+    - name: "slingshot_hsn_switch"
```

Comment:
model should be "slingshot_hsn_switch" as comment above.

Reply:
fixed